packages/builder/src/fee/FeeBuilder.interfaces.ts

```diff
@@ -9,6 +11,20 @@ export type FeeConfigBuilder = ConfigBuilder<
 >;
 
 export interface FeeConfigBuilderPrams {
-  asset: ChainAssetId;
+  address: string;
   api: ApiPromise;
+  chain: AnyParachain;
+  feeAsset: Asset;
+  transferAsset: Asset;
+}
```

Comment:
In v3 I would like to have builder interfaces as close as possible to each other. So please check other builder interfaces and think if we can improve this one.